pkg/controller/directvolumemigration/stunnel.go

```diff
@@ -81,6 +107,27 @@ data:
     TIMEOUTclose = 0
 `
 
+// generateStunnelProxyConfig loads stunnel proxy configuration from app settings
+func (t *Task) generateStunnelProxyConfig() (stunnelProxyConfig, error) {
+	var proxyConfig stunnelProxyConfig
+	tcpProxyString := settings.Settings.DvmOpts.StunnelTCPProxy
+	if tcpProxyString != "" {
+		url, err := url.Parse(tcpProxyString)
+		if err != nil {
+			t.Log.Error(err, "failed to parse TCP_PROXY setting")
+			return proxyConfig, err
```

Comment:
Any reason why we aren't using liberr.Wrap here?

Reply:
Missed it by mistake, fixed now